systemd/systemd_test.go

```diff
@@ -1356,6 +1405,88 @@ func (s *SystemdTestSuite) TestUnmaskInEmulationMode(c *C) {
 		{"--root", "/path", "unmask", "foo"}})
 }
 
+func (s *SystemdTestSuite) TestListMountUnitsEmpty(c *C) {
+	s.outs = [][]byte{
+		[]byte("\n"),
+	}
+
+	sysd := New(SystemMode, nil)
+	units, err := sysd.ListMountUnits("some-snap", "")
+	c.Check(units, IsNil)
```

Comment:
I have better luck when I expect a list to be empty to use

```suggestion
	c.Check(units, HasLen, 0)
```

otherwise I will inevitably end up refactoring the function to return an empty list instead of a nil one and the IsNil condition is false

Reply:
Done.